packages/clima_ui/lib/utilities/weather_icon.dart

```diff
@@ -0,0 +1,38 @@
+//returns the right weather icon for the right condition.
+
```

Comment:
```suggestion
/// Returns the weather icon that corresponds to the given `iconCode`.
```

The three slashes will make this appear as documentation for the function.